autosubmit/notifications/mail_notifier.py

```diff
@@ -53,26 +65,34 @@ def notify_status_change(self, exp_id, job_name, prev_status, status, mail_to):
+        server = smtplib.SMTP(self.config.SMTP_SERVER, timeout=60)
         server.sendmail(mail_from, mail_to, message.as_string())
         server.quit()
 
+    def _attach_files(self, message, files):
+        for f in files or []:
+            with open(f, "rb") as file:
+                part = MIMEApplication(file.read(), Name = Path(f).name)
+                part['Content-Disposition'] = 'attachment; filename="%s"' % Path(f).name
+                message.attach(part)
+
+
     @staticmethod
     def _generate_message_text(exp_id, job_name, prev_status, status):
-        return f'Autosubmit notification\n' \
-               f'-------------------------\n\n' \
-                f'Experiment id:  {str(exp_id)} \n\n' \
-                + f'Job name: {str(job_name)} \n\n' \
-                f'The job status has changed from: {str(prev_status)} to {str(status)} \n\n\n\n\n' \
-                f'INFO: This message was auto generated by Autosubmit, '\
-                f'remember that you can disable these messages on Autosubmit config file. \n'
+        return f'''Autosubmit notification\n
+               -------------------------\n\n
+               Experiment id:  {str(exp_id)} \n\n
+               Job name: {str(job_name)} \n\n
+               The job status has changed from: {str(prev_status)} to {str(status)} \n\n\n\n\n
+               INFO: This message was auto generated by Autosubmit,
+               remember that you can disable these messages on Autosubmit config file. \n'''
 
     @staticmethod
     def _generate_message_experiment_status(exp_id, platform=""):
-        return f'Autosubmit notification: Remote Platform issues\n' \
-               f'-------------------------\n\n' \
-                f'Experiment id:{str(exp_id)} \n\n' \
-                + f'Platform affected:{str(platform.name)} using as host:{str(platform.host)} \n\n' \
-                f'[WARN] Autosubmit encountered an issue with an remote_platform.\n It will resume itself, whenever is possible\n If issue persist, you can change the host IP or put multiple hosts in the platform.yml' + '\n\n\n\n\n' \
-                f'INFO: This message was auto generated by Autosubmit, '\
-                f'remember that you can disable these messages on Autosubmit config file. \n'
+        return f'''Autosubmit notification: Remote Platform issues\n-------------------------\n
+               Experiment id: {str(exp_id)}
+               Logs and errors: {BasicConfig.expid_aslog_dir(exp_id)}
+               Attached to this message you will find the related _run.log files.
+
+               Platform affected: {str(platform.name)} using as host: {str(platform.host)}\n\n[WARN] Autosubmit encountered an issue with a remote platform.\n It will resume itself, whenever is possible\n If this issue persists, you can change the host IP or put multiple hosts in the platform.yml file' + '\n\n\n\n\nINFO: This message was auto generated by Autosubmit,
```

Comment:
Can we break the text like in the example above? Otherwise this line has 370 characters, which is quite long.